src/app-layer-smb.c

```diff
@@ -194,9 +194,7 @@ static AppLayerDecoderEvents *SMBGetEvents(void *tx)
 static int SMBGetEventInfoById(int event_id, const char **event_name,
     AppLayerEventType *event_type)
 {
-    *event_name = "SMB event name (generic)";
-    *event_type = APP_LAYER_EVENT_TYPE_TRANSACTION;
-    return 0;
+    return rs_smb_state_get_event_info_by_id(event_id, event_name, event_type);
```

Comment:
this can be squashed into https://github.com/OISF/suricata/pull/3937/commits/8d48836de289c085c51bd32bb51709cf4a290e27

Reply:
Will do.